src/test/watchLater.test.js

```diff
@@ -0,0 +1,24 @@
+import { screen, waitFor } from '@testing-library/react'
```

Comment:
This test is wrong, 
- Add a describe
- If its needed to test watchlater component, render WatchLAter component (line 7)
- type in the search movie input and expect the response from an api call is not a good practice in a unit test, in case its needed to test the api call, we should mock the real call and just test the call with mock data.
- In this case we should render the WatchLater component, add some mock data to the store and expect that data to be rendered. 
-